src/components/Wizard/SpotlightWizard/SpotlightLogoInput.tsx

```diff
@@ -4,12 +4,12 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { useFormContext } from "react-hook-form";
+import { useFormContext, UseFormRegister } from "react-hook-form";
 import Form from "react-bootstrap/Form";
 import Row from "react-bootstrap/Row";
 
 import FormRow from "../FormRow";
-import SpotlightWizardFormData from "../formData";
+import WizardFormData, { SpotlightWizardFormData } from "../formData";
```

Comment:
Curious why yo need to import { SpotlightWizardFormData } when WizardFormData = BaseFormData &
  (InfoBarWizardFormData | SpotlightWizardFormData);

Reply:
We're importing SpotlightWizardFormData to use it with `useFormContext` on line 28. This way, it limits the context to fields defiend on SpotlightWizardFormData, so we can't, e.g., reference fields on `InfoBarWizardFormData` or the meta fields defined in `WizardFormData`.

The top level formData.ts only exports `SpotlightWizardFormData`, `InfoBarWizardFormData`, and `WizardFormData`. The import in SpotlightButtonInput is also importing `SpotlightButtonFormData`, so we have to have two imports regardless.